### PR TITLE
Created react login modal

### PR DESCRIPTION
**Pull request by sinfran**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 3000;
```
Comment (danielanatolie) [inline server.js]: The client-side is served on port 3000 and proxies all requests to 5000, why did you change the server port?

**Review by danielanatolie — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (sinfran) [reply server.js]: Oh sorry that shouldn’t have been changed, I was trying to debug something and forgot to change it back

Comment (danielanatolie) [reply server.js]: Do you mind changing it back, so we can close the PR

Comment (sinfran) [reply server.js]: Done

**Review by danielanatolie — APPROVED** [review]

**Pull request merged**